src/Rewrite.hs

```diff
@@ -59,9 +61,12 @@ version log args@(Args _ _ _ drvContents) = do
 --------------------------------------------------------------------------------
 -- Rewrite meta.homepage (and eventually other URLs) to be quoted if not
 -- already, as per https://github.com/NixOS/rfcs/pull/45
-quotedUrls :: MonadIO m => (Text -> m ()) -> Args -> ExceptT Text m (Maybe Text)
-quotedUrls log (Args _ attrPth drvFile _) = do
-  lift $ log "[quotedUrls]"
+quotedUrls ::
+  Members '[Process.Process, File.File, Error Text, Output Text] r =>
```

Comment:
Looks like I have some studying to do to learn what this does!

Reply:
Happy to explain!